source/Preferences.cpp

```diff
@@ -222,6 +225,12 @@ void Preferences::Load()
 			previousSaveCount = max<int>(3, node.Value(1));
 		else if(node.Token(0) == "alt-mouse turning")
 			settings["Control ship with mouse"] = (node.Size() == 1 || node.Value(1));
+		else if(node.Token(0) == "Confirm 'Sell Outfits' button")
+			settings["Confirm 'Sell Outfits' button"] = (node.Size() == 1 || node.Value(1));
+		else if(node.Token(0) == "Confirm 'Sell Specials' button")
+			settings["Confirm 'Sell Specials' button"] = (node.Size() == 1 || node.Value(1));
+		else if(node.Token(0) == "Allow 'Sell Outfits' with no outfitter")
+			settings["'Sell Outfits' without outfitter"] = (node.Size() == 1 || node.Value(1));
```

Comment:
Why does this need to be explicitly stated? Won't the default handling work?

Reply:
Apparently the default handling does work.